src/main/java/inventorysetups/InventorySetupsPlugin.java

```diff
@@ -1036,10 +1029,29 @@ private boolean isNewPositionInvalid(int oldPosition, int newPosition, int size)
 
 	public List<InventorySetup> filterSetups(String textToFilter)
 	{
-		final String textToFilterLower = textToFilter.toLowerCase();
-		return inventorySetups.stream()
-			.filter(i -> i.getName().toLowerCase().contains(textToFilterLower))
-			.collect(Collectors.toList());
+    return inventorySetups.stream().filter(i -> setupContains(i, textToFilter.toLowerCase())).collect(Collectors.toList());
+	}
+
+	private static boolean setupContains(InventorySetup i, String textToFilterLower)
```

Comment:
instead of `setupContains`, how about `shouldFilterSetup`?

Reply:
I prefer `setupContains` because we're filtering it in both cases, just in two different ways depending on the search text.

Reply:
`setupContains` just sounds like an incomplete sentence to me. Either way this function is testing if we should filter the current setup? Maybe `shouldDisplayCurrentSetup`?

Reply:
Yep, you're right. Okay, `shouldDisplaySetup` 👍🏼